### PR TITLE
python2Packages.plyplus: init at 0.7.5

### PR DESCRIPTION
**Pull request by Twey**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by FRidh — DISMISSED** [review]

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -8766,6 +8766,8 @@ in {
 
   ply = callPackage ../development/python-modules/ply { };
 
+  PlyPlus = callPackage ../development/python-modules/PlyPlus { };
```
[the remark this answers is not in the document]
Comment (Twey) [reply pkgs/top-level/python-packages.nix]: Do we not want to match the PyPI name, as with `PyChromecast`, `Pmw`, `PyWebDAV`, `APScheduler`, `CommonMark`, …?  I find it much less confusing if I can just drop the name of a Python package into my derivation without having to look up what name transformations have been done on it (`plyplus`? `plyPlus`?).

[the remark this answers is not in the document]
Comment (FRidh) [reply pkgs/top-level/python-packages.nix]: No

**Comment by Twey** [comment]
Is this okay?

**Comment by Twey** [comment]
Hello?

**Comment by FRidh** [comment]
folder name should also follow the normalized form, so lowercase.

**Comment by Twey** [comment]
Done!
Is this the correct normalized form?  Do we normalize to `plyplus`, `plyPlus`, `ply-plus`, or something else?  Is this documented somewhere?

**Comment by Twey** [comment]
Done!
Is this the correct normalized form?  Do we normalize to `plyplus`, `plyPlus`, `ply-plus`, or something else?  Is this documented somewhere?

**Comment by Twey** [comment]
Hello?

**Comment by Twey** [comment]
Rebased to modern master.

**Comment by samueldr** [comment]
Built fine locally, can't see anything wrong, and all asked-for changes were made.

Let's see how it copes on other platforms, as a formality.

@GrahamcOfBorg build python2Packages.plyplus

**Comment by Twey** [comment]
Thank you @samueldr!

**Pull request merged**